source/_posts/2016-07-24-top-fashion-accessories-for-men.markdown

```diff
@@ -0,0 +1,33 @@
+# Top Fashion Accessories for Men
+
+Most of the people will concur with the epigram "women's fashion is much more diverse than men!!". But who says accessories are for women only? Here are top 10 accessories that can add flavour and color to any outfit to fit the preferences of any men out there.
+
+## 1. Watch
+A wrist watch is the most conventional accessory a man can invest in. it adds a punch of simplicity as well as style to your personality. it is one of the most acceptable men's accessory. Mechanical, chronograph, leather strap watches are the latest trends this season.
+
+## 2. Tie
+A tie is pretty much one accessory that adds life to any men's costume. Various mutants include bow-tie, necktie, ascots. A tie radiates class and confidence. Pair up your suit with multicolor, printed , textured, silk ties this summer.
```

Comment:
A tie is pretty much one accessory that adds life to any men's costume. (incorrect grammatically, please reframe the line)